core/src/main/scala/org/broadinstitute/dsde/rawls/user/UserService.scala

```diff
@@ -83,6 +84,9 @@ class UserService(protected val userInfo: UserInfo, val dataSource: SlickDataSou
   def AdminAddLibraryCurator(userEmail: RawlsUserEmail) = asFCAdmin { addLibraryCurator(userEmail) }
   def AdminRemoveLibraryCurator(userEmail: RawlsUserEmail) = asFCAdmin { removeLibraryCurator(userEmail) }
 
+  val dmTemplatePath = dmConfig.getString("templatePath")
+  val dmProject = dmConfig.getString("projectID")
```

Comment:
we should not be parsing config here. Instead of passing in a `Config` object in the constructor pass in a case class.